doc/paper/markdown/outline.tex.md

```diff
@@ -0,0 +1,1293 @@
+safe and expressive, and its type system and data model naturally
+support the parallel agreement/execution strategies that make scalable.
+Move is an open-source programming language for building smart contracts
+originally developed at Facebook for the Diem blockchain. The language
+is platform-agnostic, and in addition to being adopted by , it has been
+gaining popularity on other platforms (e.g., 0L, StarCoin).
+
+In this section we will discuss the main features of the Move language
+and explain how it is used to create and manage assets on . A more
+thorough explanation of Move's features can be found in the Move
+Programming Language book[^1] and more -specific Move content can be
+found in the Developer Portal[^2], and a more formal description of Move
+in the context of Sui can be found in
+Section [\[sec:model\]](#sec:model){reference-type="ref"
+reference="sec:model"}.
+
+## Overview {#sec:move-overview}
+
+'s global state includes a pool of programmable objects created and
+managed by Move *packages* that are collections of Move modules (see
+Section [2.1.1](#sec:modules){reference-type="ref"
+reference="sec:modules"} for details) containing Move functions and
+types. Move packages themselves are also objects. Thus, objects can be
+partitioned into two categories:
+
+-   **Struct data values**: Typed data governed by Move modules. Each
+    object is a struct value with fields that can contain primitive
+    types (e.g. integers, addresses), other objects, and non-object
+    structs.
+
+-   **Package code values**: a set of related Move bytecode modules
+    published as an atomic unit. Each module in a package can depend
+    both on other modules in that package and on modules in previously
+    published packages.
+
+Objects can encode assets (e.g., fungible or non-fungible tokens),
+*capabilities* granting the permission to call certain functions or
+create other objects, "smart contracts" that manage other assets, and so
+on--it's up to the programmer to decide. The Move code to declare a
+custom object type looks like this:
+
+``` {.Move basicstyle="\\scriptsize\\ttfamily" language="Move"}
+struct Obj has key {
+  id: VersionedID, // globally unique ID and version
+  f: u64 // objects can have primitive fields
+  g: OtherObj // fields can also store other objects
+}
+```
+
+All structs representing objects (but not all Move struct values) must
+have the field and the ability[^3] indicating that the value can be
+stored in 's global object pool.
+
+### Modules {#sec:modules}
+
+A Move program is organized as a set of modules, each consisting of a
+list of struct declarations and function declarations. A module can
+import struct types from other modules and invoke functions declared by
+other modules.
+
+Values declared in one Move module can flow into another--e.g., module
+in the example above could be defined in a different module than the
+module defining . This is different from most smart contract languages,
```

Comment:
```suggestion
module defining. This is different from most smart contract languages,
```

Reply:
this looks like an incomplete sentence - could I do an edit pass on the source doc before converting to .md?